SPECS/openssh/openssh.spec

```diff
@@ -127,16 +133,17 @@ if [ $1 -eq 0 ] ; then
     fi
 fi
 
-%clean
+%{clean}
```

Comment:
The linter introduced a bug here- %clean should not have brackets. This is updated in the latest version of the tool.